docroot/modules/humsci/hs_field_helpers/src/Plugin/Field/FieldWidget/DateTimeYearOnly.php

```diff
@@ -115,6 +115,8 @@ protected function getOptions() {
    *   The year of the datetime.
    */
   protected function getYear($dateTime) {
+    // rewrite YYYY as YYYY-06-01 for year before 1969
+    if((int)$dateTime > 0 && (int)$dateTime < 1970) $dateTime = $dateTime . '-06-01';
```

Comment:
```suggestion
    if ((int) $dateTime > 0 && (int) $dateTime < 1970) $dateTime = $dateTime . '-06-01';
```
BTW don't let @jbickar  see this oneliner. I bet he'd have a stroke. :rofl:

Reply:
😄

I will have to get used to `if (` vs `if(`